ajax/libs/hyperform/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "hyperform",
+  "filename": "hyperform.min.js",
+  "version": "0.8.10",
+  "description": "Capture form validation back from the browser",
+  "homepage": "https://hyperform.js.org/",
+  "author": {
+    "name": "Manuel Strehl",
+    "url": "http://www.manuel-strehl.de"
+  },
+  "keywords": [
+    "form",
+    "forms",
+    "input",
+    "html5",
+    "form validation",
+    "validation"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperform/hyperform.git"
+  },
+  "license": "MIT",
+  "npmName": "hyperform",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": "*.js"
```

Comment:
Currently there are exactly 6 JS files in the `dist` folder. I do not intend to change that.

OTOH I know how this works usually. "Oh, we never add sub-folders to `dist`!" "Oh, we need sub-folders in `dist`!"

If you think, `**/*` is better in terms of long-term stability, I'm happy to change this :smile: .